packages/appsync-modelgen-plugin/src/__tests__/visitors/gqlv2-regression-tests/__snapshots__/appsync-dart-visitor.test.ts.snap

```diff
@@ -187,13 +189,13 @@ exports[`AppSyncDartVisitor - GQLv2 Regression Tests Works on explicit hasMany b
 // ignore_for_file: public_member_api_docs, annotate_overrides, dead_code, dead_codepublic_member_api_docs, depend_on_referenced_packages, file_names, library_private_types_in_public_api, no_leading_underscores_for_library_prefixes, no_leading_underscores_for_local_identifiers, non_constant_identifier_names, null_check_on_nullable_type_parameter, prefer_adjacent_string_concatenation, prefer_const_constructors, prefer_if_null_operators, prefer_interpolation_to_compose_strings, slash_for_doc_comments, sort_child_properties_last, unnecessary_const, unnecessary_constructor_name, unnecessary_late, unnecessary_new, unnecessary_null_aware_assignments, unnecessary_nullable_for_final_variable_declarations, unnecessary_string_interpolations, use_build_context_synchronously
 
 import 'ModelProvider.dart';
-import 'package:amplify_datastore_plugin_interface/amplify_datastore_plugin_interface.dart';
+import 'package:amplify_core/amplify_core.dart' as amplify_core;
 import 'package:flutter/foundation.dart';
```

Comment:
Can we get rid of this while we're at it?

Reply:
Sure

Reply:
Removed usage of package:flutter/foundation.dart in commit: [chore(dart-visitor): remove usage of flutter/foundation package](https://github.com/aws-amplify/amplify-codegen/pull/576/commits/65fc3dacb6c10e3a3012da44799cd49697f1054d)